LOH/1.0.0/Dockerfile

```diff
@@ -18,8 +18,15 @@ RUN apt install -y curl gcc make liblzma-dev zlib1g-dev libbz2-dev wget python3.
         sh configure --prefix=/home && \
         make && make install && \
         python3 -m pip install pysam regex pandas
- 
-ENV PATH="$PATH:/home/bin/"
+
+# setup bam-readcount
+RUN apt update && apt upgrade -y && apt install -y git g++ cmake && git clone https://github.com/genome/bam-readcount && \
```

Comment:
Can probably add apt commands to second `RUN`, unless it doesn't work,perhaps also combine git clone. Also, does bam read count have releases? Perhaps pull and install like you did for bcftools. Lastly, it might be better to set the bcftools version, and possibly bamtools version as an ENV, then just reference that. That way, if you ever need to bump the version, you only need to change in one place. See https://github.com/d3b-center/bixtools/blob/master/samtools/1.9/Dockerfile for an example of what I mean